product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: "Preparing your Azure account"
+
+---
+
+BigAnimal requires you to configure your Azure subscription before you deploy your clusters. The configurations that you perform ensure that your Azure subscription is prepared to meet your clusters' requirements and resource limits, such as: 
+
+ * Are the necessary Azure Service Providers registered for your subscription?
+ * Is there a restriction on SKUs for the Standard Esv3 and Standard Dv4 VM size families?
+ * Is there a sufficient limit on the number of vCPU or Public IP addresses in your region?
+  
+!!! Note
+    Before proceeding, see [Understanding requirements in Azure](../01_preparing_azure/01_understanding_qotas_in_azure) for details on planning for your clusters' requirements and resource limits in Azure.
+
+## Check for requirements
+
+EDB recommends using the `biganimal-preflight-azure` script to check whether all requirements and resource limits are met in your subscription. However, you can also manually check the requirements using the Azure CLI or the Azure Portal.
+
+ * [Method 1: Use EDB's shell script](#method-1-use-edbs-shell-script) (recommended)
+ * [Method 2: Manually check requirements](#method-2-manually-check-requirements)
+
+### Method 1: Use EDB's shell script
+
+EDB provides a shell script called [`biganimal-preflight-azure`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/biganimal-preflight-azure.sh), which automatically checks whether requirements and resource limits are met in your Azure subscription based on the clusters you plan to deploy.
```

Comment:
This link doesn't work - the actual file is https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/biganimal-preflight-azure (no .sh). Should fix that before release, and check with @xugy99 to see if the omission of .sh was intentional!